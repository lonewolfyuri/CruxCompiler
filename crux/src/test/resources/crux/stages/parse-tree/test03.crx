// test type

var a: void;
var b: int;
var c: float;
var d: bool;
