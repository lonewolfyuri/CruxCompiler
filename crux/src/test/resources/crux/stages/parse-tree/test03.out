program
  declarationList
    declaration
      variableDeclaration
        type
    declaration
      variableDeclaration
        type
    declaration
      variableDeclaration
        type
    declaration
      variableDeclaration
        type
